TestResources_v3/Mock Network/RequestRecorderURLProtocol.swift

```diff
@@ -73,6 +69,10 @@ class RequestRecorderURLProtocol: URLProtocol {
     }
     
     private static func record(request: URLRequest) {
+        guard latestRequest == nil else {
+            print("Reqeust for \(String(describing: currentSessionId)) already recoded. Skipping.")
```

Comment:
IMHO we should use our `logger` for tests too, so that we can disable logging (eg for CI) for (albeit incremental) test performance. Also, CI `print`s pollute CI logs

Reply:
OK, we can 👍

Reply:
Fixed in cd8dd79, I also change other existing prints in tests (one) to log.